{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/apps/careers/admin.py

```diff
@@ -42,6 +43,10 @@ class CareerAdmin(SortableModelAdmin, PageBaseAdmin):
         ('Applying', {
             'fields': ['email_address', 'application_url'],
         }),
+        ('Meta Fields', {
```

Comment:
This should be in 'Sentence case', never 'Title Case', though you may want to think of a better name for this fieldset.